### PR TITLE
Create new structure

### PR DESCRIPTION
**Pull request by tstaubitz**
- Created the new document structure
- Moved all contents of the old structure into their new location
- Added LTI changes from Sebastian's branch
- TODO: reorganize image locations

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -1,20 +1,95 @@
...
+  - Item Types:
+    - 'Video': 'features/itemtypes/video.md'
+    - 'Quiz': 'features/itemtypes/quiz.md'
+    - 'Surveys': 'features/itemtypes/surveys.md'
+    - 'Text': 'features/itemtypes/text.md'
+    - 'LTI': 'features/itemtypes/lti.md'
+    - 'Peer Assessment': 'features/itemtypes/peerassessment.md'
+   - Discussion Forum: 'features/discussionforum.md'
+   - Collab spaces: 'features/collabspaces.md'
+   - Gamification: 'features/gamification.md'
```
Comment (MrSerth) [inline mkdocs.yml:18]: The indention level is invalid. I assume you want to remove one space per line.

```suggestion
  - Discussion Forum: 'features/discussionforum.md'
  - Collab spaces: 'features/collabspaces.md'
  - Gamification: 'features/gamification.md'
```

File: mkdocs.yml [hunk 2]
```diff
@@ -1,20 +1,95 @@
 site_name: Guidelines and Documentation
 pages:
```
Comment (MrSerth) [inline mkdocs.yml:2]: The 'pages' configuration option has been deprecated and will be removed in a future release of MkDocs. Use 'nav' instead. 

```suggestion
nav:
```

File: mkdocs.yml [hunk 3]
```diff
@@ -1,20 +1,95 @@
...
+  - Community Management: 'bestpractices/communitymanagement.md'
+  - End of Course: 'bestpractices/endofcourse.md'
+  - Checklist: 'bestpractices/checklist.md'
+  - Project Management Gantt Chart: 'bestpractices/projectmanagement.md'
+- Course Administration:
+  - Setup Course Properties: 'courseadministration/courseproperties.md'
+  - Grant Permissions: 'courseadministration/permissions.md'
+  - Add Content: 
+    - Sections: 'courseadministration/addcontent/sections.md'
+    - Items:
```
Comment (MrSerth) [inline mkdocs.yml:69]: The following pages exist in the docs directory, but are not included in the "nav" configuration:
  - courseadministration/addcontent/items/item.md

File: mkdocs.yml [hunk 4]
```diff
@@ -1,20 +1,95 @@
...
+      - Hands-on: 'bestpractices/recommendedcoursecycles/scheduled/handson.md'
+      - Workshop: 'bestpractices/recommendedcoursecycles/scheduled/workshop.md'
+      - Nutshell: 'bestpractices/recommendedcoursecycles/scheduled/nutshell.md'
+    - 'Self-paced': 'bestpractices/recommendedcoursecycles/selfpaced.md'
+    - 'Blended': 'bestpractices/recommendedcoursecycles/blended.md'
+  - Prepare Material: 'bestpractices/preparematerial.md'
+  - Quiz Design: 'bestpractices/quizdesign.md'
+  - Community Management: 'bestpractices/communitymanagement.md'
+  - End of Course: 'bestpractices/endofcourse.md'
+  - Checklist: 'bestpractices/checklist.md'
```
Comment (MrSerth) [inline mkdocs.yml:62]: A relative path to 'bestpractices/checklist.md' is included in the 'nav' configuration, which is not found in the documentation files

File: mkdocs.yml [hunk 5]
```diff
@@ -1,20 +1,95 @@
...
+    - Regrading: 'courseadministration/peerassessmentmanagement/regrading.md'
+- Platform Administration:
+  - Create Course: 'platformadministration/createcourse.md'
+  - Static Pages: 'platformadministration/staticpages.md'
+- Glossary: 'glossary.md'
+  
+
+
+
 - Teacher Guidelines:
```
Comment (MrSerth) [inline mkdocs.yml:92]: What about this headline and the following lines? Did you forget to remove those?

And as I cannot comment to last two lines:

 A relative path to '22_checklist.md' is included in the 'nav' configuration, which is not found in the documentation files 
A relative path to '23_templates.md' is included in the 'nav' configuration, which is not found in the documentation files

File: mkdocs.yml [hunk 6]
```diff
@@ -1,20 +1,95 @@
...
-  - 'Granting permissions to users': '04a_course-permissions.md'
-  - 'Adding your content to the platform': '05_adding-content.md'
-  - 'Quiz Design': '06-quiz-design.md'
-  - 'Adding quizzes and questions': '07_adding-quizzes-and-questions.md'
-  - 'Quiz Submissions': '08_quiz-submissions.md'
-  - 'Peer Assessment': '10_peer-assessment.md'
-  - 'LTI': '11_lti.md'
-  - 'Gamification': '14_gamification.md'
-  - 'Static Pages': '15_static_pages.md'
   - 'Community Management': '19_community-management.md'
```
Comment (MrSerth) [inline mkdocs.yml:93]: A relative path to '19_community-management.md' is included in the 'nav' configuration, which is not found in the documentation files

File: mkdocs.yml [hunk 7]
```diff
@@ -1,20 +1,95 @@
...
   - 'End of Course': '20_end-of-course.md'
```
Comment (MrSerth) [inline mkdocs.yml:94]: A relative path to '20_end-of-course.md' is included in the 'nav' configuration, which is not found in the documentation files

File: mkdocs.yml [hunk 8]
```diff
@@ -1,20 +1,95 @@
...
   - 'Communication Matrix': '21_communication-matrix.md'
```
Comment (MrSerth) [inline mkdocs.yml:95]: A relative path to '21_communication-matrix.md' is included in the 'nav' configuration, which is not found in the documentation files

File: docs/bestpractices/quizdesign.md [hunk 9]
```diff
@@ -1,4 +1,4 @@
-![HPI Logo](img/HPI_Logo.png)
+![HPI Logo](../img/HPI_Logo.png)
 # Quiz Design
 ## General Remarks
```
Comment (MrSerth) [inline docs/bestpractices/quizdesign.md:3]: I cannot comment below, but would like to put it here:

 Documentation file 'bestpractices/quizdesign.md' contains a link to 'bestpractices/(https:/www.utexas.edu/academic/ctl/assessment/iar/students/plan/method/exams-mchoice-write.php)' which is not found in the documentation files.

The double paranthesis are too much. It should be `[good advice](https://www.utexas.edu/academic/ctl/assessment/iar/students/plan/method/exams-mchoice-write.php)`

File: docs/features/gamification.md [hunk 10]
```diff
@@ -1,4 +1,4 @@
-![HPI Logo](img/HPI_Logo.png)
+![HPI Logo](img/../HPI_Logo.png)
```
Comment (MrSerth) [inline docs/features/gamification.md:1]: Documentation file 'features/gamification.md' contains a link to 'features/HPI_Logo.png' which is not found in the documentation files.

**Comment by MrSerth** [comment]
Another remark: I find a name such as `recommendedcoursecycles` difficult to read and would either use something like CamelCase or underscores / dashes to differentiate (e.g. `recommended_course_cycles/scheduled/hands-on.md`).

As I found out, the navigation on the left side works with the proposed structure. However, it will be expanded by default as shown below. This is a known bug in MkDocs and no easy fix is available yet.
Bildschirmfoto 2020-07-13 um 18 02 45

File: mkdocs.yml [hunk 11]
```diff
@@ -1,20 +1,95 @@
...
+  - Setup Certificate Templates: 'courseadministration/certificatetemplates.md'
+  - Setup Learning Objectives: 'courseadministration/learningobjectives.md'
+  - Quiz Management: 
+    - Grant Additional Attempts: 'courseadministration/quizmanagement/additionalattempts.md'
+    - Regrading: 'courseadministration//quizmanagement/regrading.md'  
+  - Peer Assessment Management: 
+    - Grant Additional Attempts: 'courseadministration/peerassessmentmanagement/additionalattempts.md'
+    - Conflicts: 'courseadministration/peerassessmentmanagement/conflicts.md'
+    - Regrading: 'courseadministration/peerassessmentmanagement/regrading.md'
+- Platform Administration:
```
Comment (MrSerth) [inline mkdocs.yml:84]: Do you really want to add that here? I thought this is something we want to move to our Confluence?

File: mkdocs.yml [hunk 12]
```diff
@@ -1,20 +1,95 @@
...
+      - 'QC Alerts': 'features/analytics/dashboards/qcalerts.md'
+      - 'Social Shares': 'features/analytics/dashboards/socialshares.md'
+      - 'Referrer': 'features/analytics/dashboards/referrer.md'
+    - Progress: 'features/analytics/progress.md'
+  - Certificates: 'features/certificates.md'
+  - Learning Objectives: 'features/learningobjectives.md'
+  - Course Announcements: 'features/courseannouncements.md'
+  - External Tools: 
+    - CodeOcean: 'features/externaltools/codeocean.md'
+    - TeamBuilder: 'features/externaltools/teambuilder.md'
```
Comment (MrSerth) [inline mkdocs.yml:45]: Do we want to add some information on CodeOcean / TeamBuilder in our public Teaching Team Guidelines? If yes: How much?

File: mkdocs.yml (hunk 3 above)
Comment (tstaubitz) [reply mkdocs.yml]: Link to Logo is fixed, path to items.md in mkdocs.yaml is added

File: mkdocs.yml (hunk 4 above)
Comment (tstaubitz) [reply mkdocs.yml]: restructured best practices project management

File: mkdocs.yml (hunk 5 above)
Comment (tstaubitz) [reply mkdocs.yml]: yep. it's removed now

File: mkdocs.yml (hunk 7 above)
Comment (tstaubitz) [reply mkdocs.yml]: removed all

File: docs/bestpractices/quizdesign.md (hunk 9 above)
Comment (tstaubitz) [reply docs/bestpractices/quizdesign.md:3]: The link is fixed. Next to the wrong format it also was outdated.

File: docs/features/gamification.md (hunk 10 above)
Comment (tstaubitz) [reply docs/features/gamification.md]: fixed

File: requirements.txt [hunk 13]
```diff
@@ -1 +1 @@
-mkdocs==1.1.2
```
Comment (MrSerth) [inline requirements.txt:1]: You probably want to have both lines in the requirements.txt

**Pull request merged**